packages/adminrouter/extra/src/includes/server/master.conf

```diff
@@ -271,6 +271,10 @@ location ~ ^/(slave|agent)/(?<agentid>[0-9a-zA-Z-]+)(?<url>.+)$ {
     include includes/http-11.conf;
     # Disable buffering to support streaming endpoints
     include includes/disable-request-response-buffering.conf;
+    # Disabling keepalive (adds 'Connection: close') header to the
+    # response. See https://jira.mesosphere.com/browse/DCOS-42098
```

Comment:
```
    # Disabling keepalive (adds 'Connection: close') header to the
    # response.
```
The parantheses are not right, right?

Reply:
It would be clearer to move the parenthesised comment to the end of the sentence.

However, what I find confusing is that line 268 says "Streaming endpoints require keepalive" and then a few lines later we disable it. I think the comments need to spell out more clearly what is happening here.

Reply:
@jongiddy Indeed, the comments are contradicting each other. I think we should remove the include includes/http-11.conf; directive, which was added to be able to enable keepalive setting. This, however, breaks some of the harness tests.
I have added a ToDo for myself to clean it up properly.
I have also added a ticket to track that work: https://jira.mesosphere.com/browse/DCOS-42254